cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -794,6 +795,39 @@ const char* gdv_fn_initcap_utf8(int64_t context, const char* data, int32_t data_
   *out_len = out_idx;
   return out;
 }
+
+GANDIVA_EXPORT
+int32_t gdv_fn_instr_utf8(int64_t context, const char* string, int32_t string_len,
+                         const char* substring, int32_t substring_len) {
+  if (string_len == 0 && substring_len == 0) {
```

Comment:
This code is going to run for all the rows in the table. So, it is important to avoid if-statements in the code.

How about the following:
if (substring_len == 0) { return 1; } // I see instr('', '') returning 1. So, there is no need to check for string_len

Reply:
Done